tap_shopify/streams/base.py

```diff
@@ -234,6 +240,8 @@ def get_objects(self):
                 self.update_bookmark(since_id, bookmark_key='since_id')
 
             updated_at_min = updated_at_max
+        bookmark = max(min(stop_time, max_bookmark), (stop_time - datetime.timedelta(days=date_window_size))) #pylint: disable=line-too-long
```

Comment:
```suggestion
        bookmark = max(min(stop_time,
                           max_bookmark),
                       (stop_time - datetime.timedelta(days=date_window_size)))
```
I think this is more readable. The original way makes it hard to tell what we are taking the `max` of.

Reply:
Updated the code for better readability